src/schemas/racesValidationSchema.ts

```diff
@@ -0,0 +1,38 @@
+import z from 'zod';
+
+const abilityModifierZodSchema = z.object({
```

Comment:
Aqui é interessante manter o nome do campo abilityScoreIncrease, pois o objetivo é que esse campo guarde o bonus de valor que determinada raça dá em alguma habilidade, por exemplo raça humano que ganha +1 de valor em todas as habilidades.

abilityModifierZodSchema é um bom nome porém já existe o termo abilityModifier no D&D bem como abilityScore e são coisas diferentes, abilityScore é o valor da habilidade de um personagem e abilityModifier é o modificador de habilidade que é definido de acordo com o abilityScore.

Força: 16 _abilityScore_ / +3 _abilityModifier_

Reply:
https://github.com/TableRise/tablerise-backend/pull/28/commits/bf7b4a58425d02e678369b809744a07225184bbe